go.sum

```diff
@@ -1,3 +1,5 @@
+github.com/Cyb3r-Jak3/cloudflare-go v0.42.1-0.20230417212430-cf6ed3d929d8 h1:WQ1FgYZN91jtpUbN6yUxxNq3AxbSyvamEzV1nMVjOB8=
```

Comment:
this file shouldn't need to be touched; we can bump it automatically outside of this PR once the upstream version is released.

Reply:
Yeah I did a `git add .`